src/__test__/extra/typedEvents-more.test.ts

```diff
@@ -220,7 +220,6 @@ describe("typed events should compile", () => {
             confirmedAt: new Date(event.data.confirmedAt),
           };
         default: {
-          const _: never = event;
```

Comment:
This isn't unused, it's an exhaustive check.

https://www.typescriptlang.org/docs/handbook/2/functions.html#never

Reply:
I reset the history to where it was :)